### PR TITLE
Feature/create inventory details view

### PR DESCRIPTION
**Pull request by oluyaratosin123**
- create the inventory details as in the wire-frame.
- lead to the form that allows users to add new food.
- add inventory food controller with the `new` and `create` action
- add a form to create new inventory food

### CODE REVIEW

File: app/views/inventories/show.html.erb [hunk 1]
```diff
@@ -1 +1,14 @@
-<h1> This page would be for a specific inventory </h2>
+<div class="container mt-3 d-flex flex-column align-items-center">
+  <% if @inventory.user == current_user %>
+  <h1><%= @inventory.name %></h1>
+
+  <div class="d-flex justify-content-between w-50 flex-wrap">
+    <%= link_to "Add Food", new_food_path, class: "btn btn-primary m-1" if @inventory.user == current_user %>
```
Comment (guerreiropedr0) [inline app/views/inventories/show.html.erb:6]: Great job implementing the view, but I think it's better if the link redirects to the form of the relationship between inventories and foods. That would be the `new_inventory_inventory_food(@inventory.id)`

File: app/views/inventories/show.html.erb [hunk 2]
```diff
@@ -1 +1,14 @@
...
+  </div>
+
```
Comment (guerreiropedr0) [inline app/views/inventories/show.html.erb:8]: Let's add all the foods that are in this inventory as a table view

File: app/views/inventories/index.html.erb [hunk 3]
```diff
@@ -1 +1,23 @@
-<h1> This page would be for all the listed inventories by a given user </h1>
+<div class="container mt-3">
+  <h1 class="text-center mb-3">Your Inventories</h1>
+  <% if current_user %>
+    <%= link_to "New Inventory", new_inventory_path, class: "btn btn-primary" %>
```
Comment (guerreiropedr0) [inline app/views/inventories/index.html.erb:4]: [OPTIONAL] We can move this to the right side to maintain the style of the website.

**Review by guerreiropedr0 — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123 ,

You did extremely great on this project! 👏 👏
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 

## Good Points 👍
- Descriptive Pull Request
- Descriptive commit messages
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/views/inventories/show.html.erb (hunk 2 above)
Comment (oluyaratosin123) [reply app/views/inventories/show.html.erb:8]: Hello @guerreiropedr0 , I will make this changes in the next task, thanks for the feedback

**Pull request closed without merging**